cirq/study/resolver_test.py

```diff
@@ -232,12 +232,11 @@ def test_resolve_unknown_type():
 
 def test_custom_sympy_pass_through():
     class Foo:
-        def _sympy_pass_through_(self):
-            return True
+        def _resolver_value_(self):
+            return self
 
     class Bar:
-        def _sympy_pass_through_(self):
-            return False
+        pass
```

Comment:
maybe add a case here that has `_resolver_value_` method that returns `NotImplemented`?